### ISSUE TITLE
Python 3.8 support

### ISSUE THREAD

**Issue by mgedmin** [zopefoundation/zope.testbrowser#80]
Python 3.8 removed `time.clock()`, which causes

```
Traceback (most recent call last):
...
  Module zope.testbrowser.browser, line 483, in _preparedRequest
    self.timer.start()
  Module zope.testbrowser.browser, line 1420, in start
    self.start_time = self._getTime()
  Module zope.testbrowser.browser, line 1413, in _getTime
    return time.clock()
AttributeError: module 'time' has no attribute 'clock'
```

**Comment by mgedmin** [comment]
[The 3.7 docs for time.clock](https://docs.python.org/3.7/library/time.html#time.clock) say

> Deprecated since version 3.3, will be removed in version 3.8: The behaviour of this function depends on the platform: use perf_counter() or process_time() instead, depending on your requirements, to have a well defined behaviour.

I think perf_counter() is appropriate for zope.testbrowser.  It was introduced in Python 3.3 and is not available on 2.7.

**Comment by mgedmin** [comment]
Ah, I don't see test failures on Linux because time.clock() is used only on Windows.
